sell-loot.lic

```diff
@@ -19,16 +36,19 @@ class SellLoot
 
     @settings = get_settings
     town_data = get_data('town')
-    @character_hometown = @settings.hometown
-    @hometown = town_data[@settings.hometown]
+    @character_hometown = DRC.get_town_name(args.hometown || @settings.hometown)
+    @hometown = town_data[@character_hometown]
```

Comment:
Uses the new `DRC.get_town_name(..)` method introduced by https://github.com/rpherbig/dr-scripts/pull/5256